.github/workflows/ci.yml

```diff
@@ -0,0 +1,435 @@
+name: Alfresco Community Repo CI
+
+on:
+  pull_request:
```

Comment:
it's usually good to add the filter for the default branch if there aren't reasons to avoid that